docs/codeql/query-help/index.rst

```diff
@@ -0,0 +1,22 @@
+CodeQL query help
+-----------------
+
+View the query help for the queries included in the ``code-scanning``, ``security-extended``, and ``security-and-quality`` query suites for the languages supported by CodeQL.
+
+Each query help article includes: 
+
+- A summary of key metadata for the query.
+- Information about which query suites the query is included in.
+- A link to the query in the `CodeQL repository <https://github.com/gitub/codeql>`__
+- Help information to explain what potential vulnerability the query identifies and a recommendation for how to avoid introducing the problem to your code.
+
+.. toctree::
+   :titlesonly:
```

Comment:
```suggestion
   :caption: Table of Contents
   :titlesonly:
```


The introduction currently flows over into the TOC without a gap. Maybe add some kind of caption or header, to separate the TOC from the initial summary?

----

![image](https://user-images.githubusercontent.com/42641846/100744025-12b78600-33d5-11eb-9366-62a0c8922b66.png)

Reply:
Ah yes, this does look odd. I don't think showing the full (expanded) TOC is very helpful either, so I'll fix that.